### DISCUSSION TITLE
Drag and Drop for reordering of layers

### DISCUSSION THREAD

**Discussion by timkrins** [PintaProject/Pinta#760] [Ideas]
Coming from paint.net, I have found myself trying to reorder layers by dragging them.
I think it might be a common expectation for people coming from other editors.

**Comment by cameronwhite** [comment]
From a bit of digging, it looks like some work would be needed to get `Gtk.ListView` for the layers widget to support drag-reordering - https://discourse.gnome.org/t/new-gio-liststore-drag-drop-interface/12515/14 has lots of information
